integration/js/steps/WaitForAllParticipantsToTurnVideoOff.js

```diff
@@ -0,0 +1,29 @@
+const {KiteTestError, Status, TestUtils} = require('kite-common');
+const AppWaitTestStep = require('../utils/AppWaitTestStep');
+
+class WaitForAllParticipantsToTurnVideoOff extends AppWaitTestStep {
+  constructor(kiteBaseTest, sessionInfo, remoteParticipantCount) {
+    super(kiteBaseTest, sessionInfo);
+    this.remoteParticipantCount = remoteParticipantCount;
+  }
+
+  static async executeStep(KiteBaseTest, sessionInfo, remoteParticipantCount) {
+    const step = new WaitForAllParticipantsToTurnVideoOff(KiteBaseTest, sessionInfo, remoteParticipantCount);
+    await step.execute(KiteBaseTest);
+  }
+
+  stepDescription() {
+    return 'Waiting for other participants to turn off their videos';
+  }
+
+  waitCompleteCondition() {
+    return this.test.numVideoRemoteOff.toString() === this.remoteParticipantCount;
```

Comment:
Can we re-use 'WaitForRemoteParticipantsToTurnVideoOff' ?

Reply:
I also wanted to re-use the WaitForRemoteParticipantsToTurnVideoOff but it turns out to be the waitCompleteCondition can't be met `(this.test.numVideoRemoteOff.toString() === this.numberOfParticipant )` In order not to break any existing tests, I add this new step to compare the `this.test.numVideoRemoteOff.toString() ` and the `this.remoteParticipantCount` In my case, the remoteParticipantCount is 1 and the numVideoRemoteOff is also 1.

Reply:
It looks like the logic is the same for both steps:

this.test.numVideoRemoteOff.toString() === this.numberOfParticipant

this.test.numVideoRemoteOff.toString() === this.remoteParticipantCount;

This step should be reusable, it might take some adjusting on the original test